README.md

```diff
@@ -124,6 +124,30 @@ kubectl get secret secret-to-be-created   -o jsonpath='{.data.first_friend}' | b
 
 We will add more documentation once we have the implementation for the different providers. You can find some here: https://external-secrets.io
 
+## Stability and Support Level
+
+### Internally maintained:
+
+| Provider        | Stability     | Contact | 
+| --------------- |:-------------:| -------:|
+| AWS SM          |     alpha     |         |
+| AWS PS          |     alpha     |         |
+| Hashicorp Vault |     alpha     |         |
+| GCP SM          |     alpha     |         |
+
+
+### Community maintained:
+
+| Provider        | Stability     |                       Contact                           | 
+| --------------- |:-------------:|:-------------------------------------------------------:|
+| Azure KV        |     alpha     | @ahmedmus219 @ahmedmus-1A @asnowfix @ncourbet-1A @1A-mj |
```

Comment:
Please remove the ahmedmus219 account , i mistakenly commented once with my personal account instead of my business 1A account  which should be the one used